app/code/Magento/InventoryCatalog/Api/DefaultSourceRepositoryInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryCatalog\Api;
+
+use Magento\InventoryApi\Api\Data\SourceInterface;
+
+/**
+ * Represents default source
+ *
+ * @api
+ */
+interface DefaultSourceRepositoryInterface
```

Comment:
It's better not to call this service as Repository. We have a strict pattern for Repositories and which methods should be placed there.
Here you can read more about this rules - https://github.com/magento-engcom/msi/wiki/Service-Contracts#magento-repositories-basics

Reply:
In this case, I propose to rename service to 
```
DefaultSourceResolverInterface::getId(): int
```
which will return Id of the default Source (in our case 1 )